x-pack/plugins/apm/public/components/app/settings/agent_explorer/agent_latest_version/index.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { NOT_AVAILABLE_LABEL } from '../../../../../../common/i18n';
+import { AgentName } from '../../../../../../typings/es_schemas/ui/fields/agent';
+
+export function AgentLatestVersion({
+  agentName,
+  latestVersion,
+  timedOut,
+}: {
+  agentName: AgentName;
+  latestVersion?: string;
+  timedOut?: boolean;
```

Comment:
Are there other cases in which the request doesn't time out, but errors out, that also might indicate that Kibana does not have access to the bucket?

Reply:
I'm not sure about any other error types indicating this, but since the tooltip message is now pretty generic I changed the code to show the tooltip for all the errors we receive and not only time outs (addressed in [c2cc8e0](https://github.com/elastic/kibana/pull/153643/commits/c2cc8e050e456d804465ee043db6b89b3d4a4d83))